demos/inputs/inputs.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>CSS Chassis - Inputs</title>
+	<meta name="description" content="Button skeleton for styling">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="../../dist/css/chassis.css">
+	<link rel="stylesheet" href="../demos.css">
+	<link rel="stylesheet" href="./inputs.css">
+	<!-- <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,400italic,700italic" rel="stylesheet"> -->
+</head>
+<body>
+	<h1>Inputs</h1>
```

Comment:
Might be useful to have labels and inputs together in this demo.  Also some text description about the different inputs: http://view.css-chassis.com/master/demos/buttons.html